kotsu/run.py

```diff
@@ -33,23 +34,21 @@ def run(
             each model through.
         results_path: The file path to which the results will be written to, and results from prior
             runs will be read from.
-        skip_if_prior_result: Flag, if True then will not run validation-model combinations
-            that are found in the results at given results_path. If False then all combinations
-            will be ran and any prior results in results_path will be completely overwritten.
+        force_rerun: Specify if "all" models should be re-ran and their previous results
+            overwritten. Or specify a list of model IDs for which to rerun. Or don't rerun any
+            validation-model combinations that already have results if `force_rerun=None`
+            (default).
```

Comment:
something with bullets for each of the options might be easier to read but up to you, current is fine too

eg
```
        force_rerun: Argument to force models to rerun on all validations. Model-validation combinations without
            results will always be run, as well as models that are forced via this argument, which overwrites previous 
            results.
            - if `force_rerun` = "all", force re-run all models
            - if `force_rerun` = list of string ids, force re-run only these specified models
            - if `force_rerun` = None, don't force rerun any models (default)
 ```

Reply:
much better ty 3a809ec